openhands_resolver/send_pull_request.py

```diff
@@ -121,14 +121,25 @@ def make_commit(repo_dir: str, issue: GithubIssue, issue_type: str) -> None:
             check=True,
         )
         print("Git user configured as openhands")
-
+    
     result = subprocess.run(
         f"git -C {repo_dir} add .", shell=True, capture_output=True, text=True
     )
     if result.returncode != 0:
         print(f"Error adding files: {result.stderr}")
         raise RuntimeError("Failed to add files to git")
 
+    status_result = subprocess.run(
+        f"git -C {repo_dir} status --porcelain",
+        shell=True,
+        capture_output=True,
+        text=True,
+    )
+
+    if not status_result.stdout.strip():
+        raise RuntimeError("ERROR: Openhands failed to make code changes.")
```

Comment:
Could we print out any error messages from the git command here?

Reply:
added a print statement